sensor/common/compliance/nodescan_handler_test.go

```diff
@@ -116,7 +115,7 @@ func (s *NodeScanHandlerTestSuite) TestHandlerRegularRoutine() {
 	s.NoError(h.Stopped().Wait())
 }
 
-func (s *NodeScanHandlerTestSuite) TestHandlerStoppedError() {
+func (s *NodeScanHandlerTestSuite) TestHandlerStopIgnoresError() {
```

Comment:
I think `TestHandlerStopIgnoresError` test became redundant and can be removed. Would it be ok with you @vikin91 ?